www/updateUes.php

```diff
@@ -6,20 +6,122 @@
  * @author Pavel Vyskočil <vyskocilpavel@muni.cz>
  */
 
+use Jose\Component\Checker\ClaimCheckerManager;
+use Jose\Component\Checker;
+use Jose\Component\Core\AlgorithmManager;
+use Jose\Component\KeyManagement\JWKFactory;
+use Jose\Component\Signature\Algorithm\RS512;
+use Jose\Component\Signature\JWSVerifier;
+use Jose\Component\Checker\HeaderCheckerManager;
+use Jose\Component\Checker\AlgorithmChecker;
+use Jose\Component\Signature\JWSTokenSupport;
+use Jose\Component\Signature\Serializer\CompactSerializer;
+use Jose\Component\Signature\Serializer\JWSSerializerManager;
+use SimpleSAML\Configuration;
 use SimpleSAML\Logger;
 use SimpleSAML\Module\perun\Adapter;
+use SimpleSAML\Module\perun\DatabaseConnector;
 
 $adapter = Adapter::getInstance(Adapter::RPC);
 
 $entityBody = file_get_contents('php://input');
-$body = json_decode($entityBody, true);
+$token = json_decode($entityBody, true);
```

Comment:
missing input validation (at least `if($entity_body)` and `if($token !== false)`)